blackbox/docs/sql/administration/privileges.txt

```diff
@@ -94,7 +94,7 @@ and functions of the ``doc`` schema::
 .. note::
 
     In CrateDB schemas are just namespaces that are created and dropped implicitly.
-    Therefor, when applied on a SCHEMA level ``GRANT``, ``DENY`` and ```REVOKE``
+    Therefore, when ``GRANT``, ``DENY`` or ```REVOKE`` are applied on a schema level
```

Comment:
I would say "are invoked at the schema level" ( rather than "are applied" ) - as these are statements, not functions.